images/packer/packer.json

```diff
@@ -0,0 +1,103 @@
+{
+  "variables": {
+    "aws_access_key": "",
+    "aws_secret_key": "",
+    "build_version": null,
+    "kubernetes_version": "1.11.2-00",
+    "kubernetes_cni_version": "0.6.0-00",
```

Comment:
Ideally, I'd like to see these variables default to "" and have the Ansible roles detect the latest available stable version, with these variables only being used for explicitly overriding the defaults. Other wise we're going to have to continually and explicitly rev this for every release.

Reply:
If we opt for those value to be empty, should we change the `ami_name` and the tags so they don't include the Kubernetes version?

Reply:
doh, no, we still want to set those, otherwise we'll lose the ability to filter images by version.

Reply:
Sounds good, I'll leave these in then and make the ansible default to `latest`.